### DISCUSSION TITLE
Will typst have journal templates like IEEETrans?

### DISCUSSION THREAD

**Discussion by ghost** [typst/typst#869] [Ideas]
typst is awesome!

**Comment by Enivex** [comment]
Making a template that imitates the look of a specific journal isn't that hard, but that doesn't mean they'll accept typst submissions. I expect it to take years.

In the meantime the best bet is to hope that a decent way of converting typst to latex will be available.

**Comment by xkevio** [comment]
There are templates though and one of them is a recreation of IEEE papers. I don't know how close it gets to the IEEETrans class but it looks close enough to me. They are either available in the webapp or in the template repository here https://github.com/typst/templates
